tests/package_test.py

```diff
@@ -65,3 +65,9 @@ def test__package_parsing(good_package, buffer, signal):
     assert pkg.as_buffer() == buffer
     assert list(pkg.as_np()) == list(signal)
     assert pkg.blob == good_package.SerializeToString()
+
+
+def test__scale_factor(good_package, signal):
+    """Should return scaled data"""
+    pkg = DriftDataPackage(good_package.SerializeToString())
+    assert len(pkg.as_np(scale_factor=1)) == int(len(signal) / 2)
```

Comment:
Please add test for multiple scale factors. Just testing with `scale_factor=1` is very little.

Reply:
scale_factor is a part of WaveletBuffer's functionality and tested there. Here we must make sure that we pass the parameter to it,